Sources/SwiftGenCLI/templates/coredata/swift5.stencil

```diff
@@ -20,6 +20,7 @@ import {{ import }}
 {% for name, entity in model.entities %}
 {% set superclass %}{{ model.entities[entity.superEntity].className|default:"NSManagedObject" }}{% endset %}
 {% set entityClassName %}{{ entity.className|default:"NSManagedObject" }}{% endset %}
+{% set final %}{% if entity.isAbstract %}{% else %}final {% endif %}{% endset %}
```

Comment:
Avoid setting a property if you're only going to use it once, especially for for simple checks.